addGradientInfill.py

```diff
@@ -53,45 +59,62 @@ def dist(segment, point):
     y = segment.point1.y + u * py
     dx = x - point.x
     dy = y - point.y
-    return (dx*dx + dy*dy) ** .5
+
+    return (dx * dx + dy * dy) ** .5
+
```

Comment:
I just like my return statements separated a bit so you can immediately see that a function returns something.